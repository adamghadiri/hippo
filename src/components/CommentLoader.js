import React from "react";
import ContentLoader from "react-content-loader";

export default () => (
  <div className="list-item">
    <ContentLoader
      height="60"
      speed={2}
      primaryColor="#f3f3f3"
      secondaryColor="#e2e2e2"
    >
      <rect x="0" y="10%" rx="1" ry="1" width="25%" height="3" />
      <rect x="0" y="40%" rx="1" ry="1" width="85%" height="3" />
      <rect x="0" y="60%" rx="1" ry="1" width="80%" height="3" />
      <rect x="0" y="80%" rx="1" ry="1" width="40%" height="3" />
    </ContentLoader>
  </div>
);
