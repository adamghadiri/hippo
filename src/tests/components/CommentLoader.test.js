import React from "react";
import { shallow } from "enzyme";
import CommentLoader from "../../components/CommentLoader";

test("should render comment loader correctly", () => {
  const wrapper = shallow(<CommentLoader />);
  expect(wrapper).toMatchSnapshot();
});
