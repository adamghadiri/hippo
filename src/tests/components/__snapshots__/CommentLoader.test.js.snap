// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render comment loader correctly 1`] = `<_class />`;
